microsoft-edge/toc.yml

```diff
@@ -1451,6 +1451,12 @@
                 - name: WebView
                   uid: 'WebView2Script!WebView:class'
 
+            - name: HoloLens 2
+              items:
+                - name: Unity Plugin
+                  href: /windows/mixed-reality/develop/advanced-concepts/webview2/webview2-unity-plugin
```

Comment:
@Reezaali - pls review whether this design will work, linking to a different doc-set directly from the toc.yml.  See toc.yml file > wv2 > Reference, it links to various locations.
@peiche-jessica 
Live:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/webview2-api-reference

eg here's a similar entry:
```
            - name: .NET
              items:
                - name: Microsoft.Web.WebView2.Core
                  href: /dotnet/api/microsoft.web.webview2.core
```